From 116c56932e0f5578c905bdf4d2d67817367e4df7 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Fri, 5 Jun 2015 16:23:35 -0500
Subject: [PATCH] neutron sudoers entry

---
 neutron.sudoers | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 neutron.sudoers

diff --git a/neutron.sudoers b/neutron.sudoers
new file mode 100644
index 0000000..92411ee
--- /dev/null
+++ b/neutron.sudoers
@@ -0,0 +1 @@
+neutron ALL = (root) NOPASSWD: ALL
-- 
2.5.5

